### DISCUSSION TITLE
Error experience when using file_base in [Outputs] block

### DISCUSSION THREAD

**Discussion by duenam5913** [idaholab/moose#18323] [Q&A General] [answered]
I'm currently using BISON which has been updated and rebuilt 3 weeks ago. I noticed that if I include the input parameter "file_base" in the [Outputs] block with an string such as "file_base=out" that following error occurs:
```
*** ERROR ***
An output file with the name, out.csv, already exists.
```
I verified that there are no files named as out.csv in the directory as well. The code is able to run successfully the input parameter "file_base" was not included. 

Has anyone else encountered an issue like this using "file_base" in the [Outputs] block?

**Comment by GiudGiud** [comment, marked as answer]
'out' might be a special case (bug) because that's the default for the suffix.
Could you please try another file_base?

**Reply by duenam5913** [reply]
Thank you for your reply. I tried another case with "file_base=test_case" and "file_base=try_2" but encountered the same error for both.

**Reply by GiudGiud** [reply]
Could you please paste your [Outputs] block here? Maybe there is some duplicates (like two csv outputs)

**Reply by GiudGiud** [reply]
This will trigger the error for example

```
[Outputs]
  csv = true

  [csv]
    type = CSV
  []
[]

**Reply by GiudGiud** [reply]
Ok I m creating an issue for this thing. Please let us know if this is not your problem

**Reply by aeslaughter** [reply]
Perhaps we need to improve the message, but @GiudGiud "csv=true" is equivalent to the block you added. It is even documented: https://mooseframework.inl.gov/syntax/Outputs/index.html

**Reply by aeslaughter** [reply]
Another option would be to change the name of the block created by "csv=true" some something else, e.g., "_csv" or "auto_csv", etc. However, this has been this way for 5+ years, so I don't think we should sink a lot of time into it. Perhaps, just a better error message is the solution.

**Reply by duenam5913** [reply]
This is the output block that I am using. If the file_base line is commented out than no error occurs. Is there something with the input parameters of the [no_print] block that should be adjusted then?

```
[Outputs]
  interval = 1
  exodus = false
  csv = true
  print_linear_residuals = true
  perf_graph = true
  color = false
  [console]
    type = Console
    max_rows = 25
  []
  [exodus]
    type = Exodus
    interval = 3
  []
  [no_print]
    type = CSV
    enable = false
  []
  file_base = test_case
[]
```

**Reply by GiudGiud** [reply]
just remove the csv = true line. 
Same for exodus. Why have exodus = false then an exodus block ?

**Reply by duenam5913** [reply]
Thank you for the assistance, that fixed the error in my input deck. I've removed the exodus=false line as well since it was unnecessary.
